### ISSUE TITLE
Windows Security identifies Textify as a threat?

### ISSUE THREAD

**Issue by Kaedin** [m417z/Textify#4]
Hi Dev,

Just tried to download and install Textify on Windows 1909 with Windows Security enabled (with OS default configuration). However, the executable getting identified as a threat by both Edge (Chromium) and Windows Security:

![image](https://user-images.githubusercontent.com/26382655/74135481-091a2300-4bf5-11ea-9448-11b6db300a61.png)

Could you confirm whether this is a false positive and would there be some resolution other than whitelisting this application?

Best regards,
Kaedin

**Comment by dartraiden** [comment]
> would there be some resolution

https://www.microsoft.com/wdsi/filesubmission

**Comment by m417z** [comment]
Hi Kaedin,

It's a false positive. The latest version of Textify was released a couple of days ago, and unfortunately it's common for AVs to flag it until it becomes a bit more popular.

I've contacted Microsoft, and [according to VirusTotal](https://www.virustotal.com/gui/file/55faaaacc49e916a5d4fdd24e657b80a7cd675c55c16b2ddd936051f1f978d25/detection) they removed the detection.

See also my comments [here](https://rammichael.com/textify/comment-page-2#comment-8470).
